### DISCUSSION TITLE
Why yazi main repo is not a part of `yazi-rs` organization?

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [sxyazi/yazi#1340] [1. Q&A]
Out of curiosity, it would be nice to have all yazi-related repos in one place like: https://github.com/yazi-rs

**Comment by AnirudhG07** [comment]
Well I did make [awesome-yazi](https://github.com/AnirudhG07/awesome-yazi), which I am collecting all flavours and plugins I find here and there.

Plus there are various resources already present in yazi or yazi-rs.

**Reply by wojciech-kulik** [reply]
Nice! I will take a look there

**Reply by AnirudhG07** [reply]
Sure. Feel free to add any plugins/flavors you may find I missed
